corehq/apps/hqmedia/tasks.py

```diff
@@ -214,6 +219,26 @@ def build_application_zip(include_multimedia_files, include_index_files, app,
     DownloadBase.set_progress(build_application_zip, 100, 100)
 
 
+def _check_ccz_locale_id_integrity(file_cache):
+    for path in ('default/app_strings.txt', 'suite.xml'):
+        if path not in file_cache:
+            return [_("Could not find {path} in CCZ").format(path)]
+
+    app_strings_ids = {
+        line.decode("utf-8").split('=')[0]
```

Comment:
its difficult to verify this without knowing what a `line` can be?
i can see someone unaware of how app_strings are put in the ccz being confused about this.
Would it be worth to add an example here as a comment?